### PR TITLE
Add Travis CI configuration file

### PR DESCRIPTION
**Pull request by AntonioND**
Let's see if this works.

This isn't going to help with BSD or Windows, though, but I suppose it's better than nothing.

### CODE REVIEW

**Comment by Ben10do** [comment]
It might be nice in future to run the tests also, but this is certainly looking good!

**Comment by AntonioND** [comment]
I was making the test script return an error code to run them as well.

File: .travis-deps.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if [ $TRAVIS_OS_NAME = "osx" ]; then
+    brew update
+    brew install flex bison libzip pkg-config
```
Comment (Ben10do) [inline .travis-deps.sh]: BTW, `flex` and `bison` should already be installed, so there shouldn't be any need to `brew install` these.

Also, is there any reason for `libzip`? I've not got it installed on my Mac.

Comment (AntonioND) [inline .travis-deps.sh]: That's what happens when you copy paste code... xD

I'll remove them, then.

Comment (AntonioND) [inline .travis-deps.sh]: It should be libpng.

Comment (Ben10do) [inline .travis-deps.sh]: That sounds better 😝

**Comment by AntonioND** [comment]
I think this should be enough for now. We should have a test target in the Makefile so that we don't have to modify the travis config file.

**Comment by AntonioND** [comment]
That osx build took forever to start, 30 min...

**Comment by Ben10do** [comment]
It certainly did… but at least it's succeeded.

**Pull request merged**